pom.xml

```diff
@@ -79,12 +79,27 @@
       <artifactId>xmlresolver</artifactId>
       <version>4.5.1</version>
     </dependency>
+    <dependency>
+      <groupId>xml-apis</groupId>
+      <artifactId>xml-apis</artifactId>
+      <version>1.4.01</version>
+    </dependency>
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
+      <version>2.17.0</version>
+    </dependency>
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
       <version>4.13.1</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
And this one ?

Reply:
It is used in the code base here: https://github.com/xspec/xspec-maven-plugin-1/blob/master/src/main/java/io/xspec/maven/xspecMavenPlugin/utils/CatalogWriter.java#L88